containers/vpc-mongodb/app/main.ts

```diff
@@ -0,0 +1,86 @@
+import mongoose from "mongoose";
+import { randomIntegerBetween } from "@std/random";
+
+const MONGODB_CERT_PATH = "/tmp/mongodb_cert.pem";
+
+const MONGODB_URI = Deno.env.get("MONGODB_URI")
+const MONGODB_CERT = Deno.env.get("MONGODB_CERT")
+
+let mongooseConnected = false;
+
+async function getMongooseConnection() {
+  if (!mongooseConnected) {
+    await mongoose.connect(MONGODB_URI || "", {
+      tls: true,
+      tlsCAFile: MONGODB_CERT_PATH,
+    } as mongoose.ConnectOptions);
+
+    mongooseConnected = true;
+  }
+}
+
+const CHECK_CONNECTION = new URLPattern({ pathname: "/check_connection" });
```

Comment:
For reviewers, this looks semi-horrible, but it seems to be the documented way to do routing with Deno HTTP server: https://docs.deno.com/examples/http_server_routing/